test/unit/request/request.test.js

```diff
@@ -0,0 +1,29 @@
+import Request from '../../../src/request';
+import { Client } from '../../../src/client';
+import expect from 'expect';
+
+describe('Request', function() {
+  describe('constructor', function() {
```

Comment:
Just out of curiosity, is there a way for us to mock request timeout, and test that to be the value `timeout`?